datasets/_externals/ext_abuja_mou_psc.yml

```diff
@@ -0,0 +1,64 @@
+type: external
+title: Abuja MoU Port State Control Inspections
+entry_point: zavod.runner.local_enricher:enrich
+full_dataset: abuja_mou_psc
+disabled: false
+hidden: false
+coverage:
+  schedule: "@weekly"
+  start: 2025-08-18
+exports:
+  - statistics.json
+  - entities.delta.json
+deploy:
+  memory: "5000Mi"
+load_statements: true
+ci_test: false
+summary: >-
+  Vessels inspected by member authorities under the Abuja MoU.
```

Comment:
I think we'd also like a description - normally include one in the enricher too - we actually just copy the target dataset's details. I personally think we can qualify it a bit, e.g.